go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -366,6 +366,22 @@ func TestScripts(t *testing.T) {
 		skipped = append(skipped, newFormatSkippedScripts...)
 	}
 	enginetest.TestScripts(t, newDoltHarness(t).WithSkippedQueries(skipped))
+	if !types.IsFormat_DOLT(types.Format_Default) {
+		t.Skip("not fixing unique index on keyless tables for old format")
+	}
+	enginetest.TestScript(t, newDoltHarness(t), queries.ScriptTest{
```

Comment:
Don't put this here. Add it to somewhere in `ddl_queries.go` and call it from TestDoltDdlScripts